### ISSUE TITLE
protodune SP reco2 files missing from rucio

### ISSUE THREAD

**Issue by hschellman** [DUNE/data-mgmt-ops#716]

**Comment by StevenCTimm** [comment]
This is sort of vague, can you at least give us a SAM query of the ones that you found missing?

**Comment by StevenCTimm** [comment]
OK well if a file like that is assigned to "dune" scope in metacat then it was never declared to rucio yet. .this is a bit of a surprise because at least the PDSPPRod4 reco is definitely declared to rucio.  Let me have a look.

**Comment by hschellman** [comment]
samweb list-files "run_number 5204 and data_tier reco-recalibrated with limit 1" -e dune
np04_raw_run005204_0046_dl3_reco1_13835485_0_20201109T225723Z_reco2_21365863_0_20210622T232155Z.root
[schellma@dunegpvm03 ~]$ 
[schellma@dunegpvm03 ~]$ samweb locate-file np04_raw_run005204_0046_dl3_reco1_13835485_0_20201109T225723Z_reco2_21365863_0_20210622T232155Z.root -e dune
enstore:/pnfs/dune/tape_backed/dunepro/protodune-sp/full-reconstructed/2021/detector/physics/PDSPProd4/00/00/52/04(2167@fl9103l8)

**Comment by StevenCTimm** [comment]
so the file in question is a 7GeV run 
The parent reco1 file is in rucio 
but this reco2 file is not.

I am not sure I was ever informed of the existence of this dataset before now.
It appears that we never declared the reco2 files to rucio.. as I said also the "dune" namespace in metacat contains predominantly stuff that didn't fit in any of the known rucio scopes and has not yet been declared to rucio.. you can see that it's quite big.
We'll get this eventually but not right away, there are much more important fish to fry.

**Comment by hschellman** [comment]
Yes, current data taking has priority but I thought this needed to be flagged, Maybe Jake’s CAF’s are so good nobody is even looking at this anymore.


On Sep 13, 2024, at 10:59 AM, Steven Timm ***@***.***> wrote:


[This email originated from outside of OSU. Use caution with links and attachments.]


so the file in question is a 7GeV run
The parent reco1 file is in rucio
but this reco2 file is not.

I am not sure I was ever informed of the existence of this dataset before now.
It appears that we never declared the reco2 files to rucio.. as I said also the "dune" namespace in metacat contains predominantly stuff that didn't fit in any of the known rucio scopes and has not yet been declared to rucio.. you can see that it's quite big.
We'll get this eventually but not right away, there are much more important fish to fry.
